backend/reservations/constants.py

```diff
@@ -1,9 +1,9 @@
 ROOM_LIST = {
     'Queen': [
-        'Hard Rock - Standard 2 Queens (50% fee)',
-        'Hard Rock - Lakeview 2 Queens',
-        'Hard Rock - Lakeview 2 Queens (Post Sale)',
-        'Hard Rock - Lakeview Balcony 2 Queens',
+#        'Hard Rock - Standard 2 Queens (50% fee)',
```

Comment:
If we want to use RoomBaht to allocate rooms for the Hard Rock, how would we do so?
One thought is to have room lists for each venue, and then use the --is-hard-rock flag to switch the room list.

Could you add these back into a separate constant for `HARD_ROCK_ROOM_LIST`?